### PR TITLE
Update allows for removal of dates

### PR DESCRIPTION
**Pull request by Melvin-Tan**
#73

### CODE REVIEW

**Comment by Melvin-Tan** [comment]
Halp, I have no idea why build failed. 
The exit controller breaks on appveyor.
My delete tasks individually breaks on both (but I checked that line, and run the tests locally, and it didn't cause any problems leh).

**Comment by louislai** [comment]
gimme some more time to review this

**Comment by Melvin-Tan** [comment]
Omg yaaaassss no build break :D
Shame on me, I forgot to push 1 file and some commands still use startdate/ enddate/ instead of from/ to/ by/ in the DeleteTaskCommandTest file.

File: src/test/java/guitests/UpdateTaskCommandTest.java [hunk 1]
```diff
@@ -140,6 +140,160 @@ public void updateEvents() {
     }
```
Comment (Melvin-Tan) [inline src/test/java/guitests/UpdateTaskCommandTest.java:141]: Summary of tests below this line.....
Event -> Task with deadline
Event -> Floating
Task with deadline -> Event
Task with deadline -> Floating
Floating -> Event
Floating -> Task with deadline
All of which should be updated correctly

Event -> Floating + Task with deadline
Event -> Event + Floating Task
Event -> Event + Task with deadline
Event -> All floating, task with deadline, and event
All of these should fail to update and the original event should still be around.

**Pull request merged**